core/src/main/java/ai/timefold/solver/core/impl/move/director/MoveDirector.java

```diff
@@ -122,6 +124,23 @@ public final <Entity_, Value_> void changeVariable(PlanningVariableMetaModel<Sol
         return value;
     }
 
+    @Override
+    public <Entity_, Value_> boolean isValueInRange(GenuineVariableMetaModel<Solution_, Entity_, Value_> variableMetaModel,
```

Comment:
It looks like we are not using this method as well.

Reply:
These methods are not for us. They are for users. They will be used, once people start writing their own move streams.